front/lib/resources/group_resource.ts

```diff
@@ -189,6 +199,44 @@ export class GroupResource extends BaseResource<GroupModel> {
     return new this(GroupModel, group.get());
   }
 
+  async addMember(
+    auth: Authenticator,
+    userId: string,
```

Comment:
Pretty convinced we can force this to be UserType?

Reply:
We can but we will still need to fetch the userResource because that's what's expected from `getActiveMembershipOfUserInWorkspace`, so I figured out better just pass the id?

Reply:
Well if our code is structured as expected with the UserType the usage point will be `addMember(auth, user)` vs here it will be `addMember(auth, user.sId)` 

The latter is a bit more error prone and the typing as `string` is less retrictive than `UserType` so I would go for `UserType` as it's actually simpler and more restrictively typed.

Reply:
Will add a commit before merging 👍🏻